.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
+* @mcdonnnj
```

Comment:
You shouldn't be the sole code owner.  Find someone else to include- perhaps @hillaryj?

Reply:
I have added both of you in 5614c30